samples/openthread/ncp/CMakeLists.txt

```diff
@@ -12,6 +12,8 @@ set(NRF_SUPPORTED_BOARDS
 
 set(OT_NCP_VENDOR_HOOK_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR})
 
+set(CONF_FILE "prj.conf ../openthread.conf")
```

Comment:
This is not acceptable, users should always be able to specify `-DCONF_FILE=<somefile.conf>` on command line when invoking CMake (or `west build`)